packages/sanity/src/_internal/cli/commands/migration/createMigrationCommand.ts

```diff
@@ -76,12 +76,12 @@ const createMigrationCommand: CliCommandDefinition<CreateMigrationFlags> = {
       .replace(/\s+/g, '-')
       .replace(/[^a-z0-9-]/g, '')
 
-    const destDir = path.join(MIGRATIONS_DIRECTORY, sluggedName)
+    const destDir = path.join(workDir, MIGRATIONS_DIRECTORY, sluggedName)
     if (existsSync(destDir)) {
       if (
         !(await prompt.single({
           type: 'confirm',
-          message: `Migration directory ./${destDir} already exists. Overwrite?`,
+          message: `Migration directory ${chalk.cyan(destDir)} already exists. Overwrite?`,
```

Comment:
I can see this _possibly_ being a bit confusing if you're not at the root (I would `ls` and look around for the folder). Feel free to take or leave the suggestion -- it may add noise.

```suggestion
          message: `Migration directory ${chalk.cyan(destDir)} in the root directory already exists. Overwrite?`,
```

Reply:
Sorry for the delay in replying! The full path to the directory will be printed, which hopefully eliminate any ambiguity:

```
Migration directory /Users/ash/projects/sanity/dev/test-studio/migrations/nesty-testy already exists. Overwrite?
```